pyproject.toml

```diff
@@ -3,8 +3,9 @@
 
 [tool.bandit]
 exclude_dirs = ["/venv/", "tests/integration/any_charm.py"]
+skips = ["B603", "B404", "B607"]
```

Comment:
What are these?

Reply:
Using subprocess methods without check=True (using them with that option is worse anyway), complains about using the subprocess library, executing a command with a partial path.

Reply:
Those are important items related to security, e.g., calling a binary with a partial path is dangerous since that path is context dependent so anything can happen

Reply:
The problem with removing this is that the full path is dependant on the unit's name, so it can't be specified. Do you know how could I  work around that issue while still specifying the whole path?

Reply:
I'm asking since when I specify an absolute path I still get the same bandit error when testing locally.

Reply:
You should be able to specify this by using `self.unit.name` (you may need to replace `/` with `-`).

Reply:
For some unknown reason, the check fails even if I provide the full path. Maybe it is an error from bandit?

Reply:
I just downloaded this branch, commented out that line and ran `tox -e lint && tox -e unit` and things passed fine. Can you confirm it's failing for you if you remove it?

Reply:
Now, after the last modifications, it is passing. I'll remove the line.